### DISCUSSION TITLE
Migrate from sqlite to postgresql

### DISCUSSION THREAD

**Discussion by MvrckTheBald** [drakkan/sftpgo#1306] [General]
Hi everyone,

I have successfully set up and started using an installation of SFTPGo using Docker. I am already enjoying its features.

I would like to inquire about the best approach to migrate from SQLite to PostgreSQL while preserving the configuration and user data.

Thank you in advance.

**Comment by drakkan** [comment]
Hello,

export the configuration from the WebAdmin UI, configure SFTPGo to use Postgres, import the configuration

**Comment by MvrckTheBald** [comment]
Hi drakkan, thanks for your answer. I will follow your instruction.
